### ISSUE TITLE
No spacing between multiple verses

### ISSUE THREAD

**Issue by bmalchert** [tim-hub/obsidian-bible-reference#56]
After I reference a range of verses (e.g. `--matt20:1-16`), and insert the suggested text, there is consistently no space between inserted verses. So the above reference using the ESV translation looks like this after selecting the suggestion:

```
> [!Bible] 
> “For the kingdom of heaven is like a master of a house who went out
> early in the morning to hire laborers for his vineyard.After agreeing with
> the laborers for a denarius  a day, he sent them into his vineyard. 
> 
  [matt 20:1-2 - ESV](https://bible-api-bff.bai.uno/bolls-life/ESV/40/20/)
```

Note the lack of spacing between _vineyard_ and _After_.

The expected result would be to have a whitespace of some kind between each verse in the range.

**Comment by JPT62089** [comment]
@tim-hub, This should be resolved with PR #58

**Comment by tim-hub** [comment]
> @tim-hub, This should be resolved with PR #58

Thanks so much for the PR

**Comment by tim-hub** [comment]
Hi @bmalchert Can you help to confirm whether this issue still exists or not?

After #74
